src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -2,7 +2,7 @@ import { coerceCssPixelValue } from '@angular/cdk/coercion';
 import { AfterViewInit, ChangeDetectionStrategy, ChangeDetectorRef, Component, ContentChild, ElementRef, EventEmitter, forwardRef, Input, OnChanges, OnDestroy, Output, SimpleChanges, TemplateRef, ViewChild } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 import { Subject } from 'rxjs';
-import { MenuTriggerDirective } from '../menu/menu-trigger/menu-trigger.directive';
+import { MenuTriggerDirective } from '../menu';
```

Comment:
Could this import be changed to be '../menu/index' or back to the previous value. 

TypeScript assumes an index.ts file when we just specify a folder, however `ng-packagr` can get confused and cause errors if we point to a directory instead of a file.

Reply:
I reverted that. TSLint makes it a warning, so I thought I would fix it on the way ...